server/services/database/users.js

```diff
@@ -70,6 +73,22 @@ const createUser = ({ email, password, name, role, languages }) => {
     });
 };
 
+const createUserUser = (email, password, name, role) => {
```

Comment:
I think this function is not used anymore so you can remove it from this file :)